test-harness/specs/validate-body-params/json-inference-invalid-request.pc.txt

```diff
@@ -0,0 +1,85 @@
+====test====
+When two examples of the same request were supplied to Prism
+and there is a difference between request bodies
+Prism can reject the request which is not compatible with any of example bodies
+====spec====
+{
+  "info": {
+    "name": "Test"
+  },
+  "item": [
+    {
+      "id": "3b4f5a70-2a8f-46ba-a04b-a6f80621ad3f",
+      "request": {
+        "url": {
+          "path": ["path"]
```

Comment:
Is this a valid path by the way? Shouldn't it be `/path`?

Reply:
Ney, postman stores paths as `'/path'.split('/').slice(1)`